src/commands/ehre.ts

```diff
@@ -0,0 +1,99 @@
+import {SlashCommandBuilder, SlashCommandSubcommandBuilder, SlashCommandUserOption} from "@discordjs/builders";
+
+import {Client, CommandInteraction, InteractionReplyOptions, MessagePayload} from "discord.js";
+
+import {ApplicationCommand, CommandResult} from "./command";
+import {EhreGroups, EhrePoints, EhreVotes} from "../storage/model/Ehre";
+
+
+async function createEhreTable(): Promise<MessagePayload | InteractionReplyOptions> {
+    const userInGroups = await EhrePoints.getUserInGroups();
+    return {
+        embeds: [{
+            color: 2007432,
+            author: {
+                name: "Shitpost Bot",
+                url: "https://discordapp.com/users/663146938811547660/"
+            },
+            fields: [
+                userInGroups.best ? {
+                    name: "Ehrenpate",
+                    value: userInGroups.best ? `<@${userInGroups.best}>` : "",
+                    inline: false
+                } : {
+                    name: "Fangt an",
+                    value: "Noch ist niemand geährt worden"
+                },
+                ...userInGroups.middle.length > 0 ? [{
+                    name: "Ehrenbrudis",
+                    value: userInGroups.middle.map(user => `<@${user}>`).join(","),
+                    inline: false
+                }] : [],
+                ...userInGroups.bottom.length > 0 ? [{
+                    name: "Ehrenhafte User",
+                    value: userInGroups.bottom.map(user => `<@${user}>`).join(","),
+                    inline: false
+                }] : []
```

Comment:
Kannst mal überlegen, ob man das nicht in eine Funktion auslagern könnte. Ist kein Muss aber vielleicht macht das Sinn.
Ansonsten auf jeden Fall klammern um den Spread Operator.

```suggestion
                userInGroups.best ? {
                    name: "Ehrenpate",
                    value: userInGroups.best ? `<@${userInGroups.best}>` : "",
                    inline: false
                } : {
                    name: "Fangt an",
                    value: "Noch ist niemand geährt worden"
                },
                ...(userInGroups.middle.length > 0 ? [{
                    name: "Ehrenbrudis",
                    value: userInGroups.middle.map(user => `<@${user}>`).join(","),
                    inline: false
                }] : []),
                ...(userInGroups.bottom.length > 0 ? [{
                    name: "Ehrenhafte User",
                    value: userInGroups.bottom.map(user => `<@${user}>`).join(","),
                    inline: false
                }] : [])
```